File: king-admin-java/src/main/java/com/oukingtim/domain/SysUserRole.java
package com.oukingtim.domain;

import com.baomidou.mybatisplus.annotations.TableName;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * Created by oukingtim
 */
@TableName("sys_user_role")
@Data
@EqualsAndHashCode(callSuper = false)
public class SysUserRole extends BaseModel<SysUserRole> {

    private String roleId;
    private String userId;

}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/domain/SysUserRole.kt
package com.oukingtim.domain

import com.baomidou.mybatisplus.annotations.TableName

/**
 * Created by oukingtim
 */
@TableName("sys_user_role")
class SysUserRole : BaseModel<SysUserRole>() {
    var roleId: String? = null
    var userId: String? = null
}